pkg/apis/autoscaling/register.go

```diff
@@ -83,7 +83,7 @@ const (
 	// scale-to-zero-pod-retention-period global setting.
 	ScaleToZeroPodRetentionPeriodKey = GroupName + "/scaleToZeroPodRetentionPeriod"
 
-	// MetricAggregationAlgorithm is the annotation that can be used for selection
+	// MetricAggregationAlgorithmKey is the annotation that can be used for selection
```

Comment:
ha, I bike-shedded all over the PR that introduced this comment and didn't spot this :D

Reply:
🥳